From 4f35dfae765a29819dc0bf7ac0dfdba21239f3ca Mon Sep 17 00:00:00 2001
From: "Martin T. H. Sandsmark" <sandsmark@samfundet.no>
Date: Wed, 16 May 2012 02:45:09 +0200
Subject: [PATCH] Increase max global stack size back to 1000.

Some pages have started running out of stack frames (e. g. Twitter).
Instead of reducing this the next time, consider reducing stack usage.
---
 kjs/bytecode/machine.cpp.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kjs/bytecode/machine.cpp.in b/kjs/bytecode/machine.cpp.in
index 3a2686b..a434c36 100644
--- a/kjs/bytecode/machine.cpp.in
+++ b/kjs/bytecode/machine.cpp.in
@@ -42,7 +42,7 @@
 // If we improve our stack usage, we can bump this number.
 #define KJS_MAX_STACK 100
 #else
-#define KJS_MAX_STACK 700 // ### set system specific
+#define KJS_MAX_STACK 1000 // ### set system specific
 #endif
 
 namespace KJS {
-- 
1.7.10

